.github/CODEOWNERS

```diff
@@ -130,42 +124,50 @@
 /sdk/digitaltwins/ @sjiherzig @johngallardo @olivakar
 
 # PRLabel: %Event Grid
-# AzureSdkOwners: @sarangan12 
-# ServiceLabel: %Event Grid
 /sdk/eventgrid/ @sarangan12
 
+# AzureSdkOwners: @sarangan12
+# ServiceLabel: %Event Grid
+
 # PRLabel: %Event Hubs
-# ServiceLabel: %Event Hubs
-# AzureSdkOwners: @deyaaeldeen @jeremymeng 
 /sdk/eventhub/ @deyaaeldeen @jeremymeng @HarshaNalluru
 
+# ServiceLabel: %Event Hubs
+# AzureSdkOwners: @deyaaeldeen @jeremymeng
+# ServiceOwners: @aschhabria
```

Comment:
getting this error from the linter
>   -aschhabria is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.

Reply:
That's my bad.  I pasted the MS alias rather than GH handle:  `axisc`

Reply:
He still has none of the orgs+permissions that I asked him to get, so ywe'll need to baseline him.

Reply:
@jsquire do I need to do anything to re-baseline, or I just override and merge this PR then magic happens?

Reply:
You'll need to add them to the [CODEOWNERS baseline errors file(https://github.com/Azure/azure-sdk-for-js/blob/main/.github/CODEOWNERS_baseline_errors.txt) or they'll fail the linter.

Reply:
Done!